specification/schemas/metadata_schema_v0.1.json

```diff
@@ -74,4 +74,5 @@
                 }
             }
         }
+    }
```

Comment:
I think it worked because it occurred after binding it into data. Only the bind data is used for validation.